drivers/soundwire/bus.c

```diff
@@ -846,15 +847,22 @@ static int sdw_slave_clk_stop_callback(struct sdw_slave *slave,
 				       enum sdw_clk_stop_mode mode,
 				       enum sdw_clk_stop_type type)
 {
-	int ret;
+	struct device *dev = &slave->dev;
+	struct sdw_driver *drv;
+	int ret = 0;
 
-	if (slave->ops && slave->ops->clk_stop) {
-		ret = slave->ops->clk_stop(slave, mode, type);
-		if (ret < 0)
-			return ret;
+	/*
```

Comment:
we can remove "ret variable" and return drv->ops->clk_stop(slave, mode, type); or return 0 otherwise

Reply:
done, thanks for the suggestion @ranj063